lib/src/components/motor/service.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:grpc/grpc.dart';
+
+import '../../gen/common/v1/common.pb.dart';
+import '../../gen/component/motor/v1/motor.pbgrpc.dart';
+import '../../resource/manager.dart';
+import '../../utils.dart';
+import 'motor.dart';
+
+class MotorService extends MotorServiceBase {
+  final ResourceManager _manager;
+
+  MotorService(this._manager);
+
+  Motor _motorFromManager(String name) {
+    try {
+      return _manager.getResource(Motor.getResourceName(name));
+    } catch (e) {
+      throw (GrpcError.notFound(e.toString()));
+    }
+  }
+
+  @override
+  Future<DoCommandResponse> doCommand(ServiceCall call, DoCommandRequest request) async {
+    Motor motor = _motorFromManager(request.name);
+    var result = await motor.doCommand(request.command.toMap());
+    return DoCommandResponse(result: result.toStruct());
+  }
+
+  @override
+  Future<GetPositionResponse> getPosition(ServiceCall call, GetPositionRequest request) async {
```

Comment:
I forgot what we settled on for dart, `get` or no `get`?

Reply:
we decided no `get` for the clients, but for service wrappers we decided to mimic the proto exactly because we don't have the same inheritance pattern that allowed us to change the names as in the clients.